controllers/ScraperService.js

```diff
@@ -1,7 +1,12 @@
 import fetchData from '../helpers/Fetch';
+import crawlerData from './../helpers/Crawler';
```

Comment:
`crawler` is a noun. use the verb `crawl` instead. do it for `..helpers/Crawler` also.